### DISCUSSION TITLE
Doubt about Google HSM changes

### DISCUSSION THREAD

**Discussion by carlosspohr** [ebourg/jsign#205] [Q&A] [answered]
Hi @ebourg,

You add support for Google HSM keys for java jar signature last year (issue #177), and today I received a reminder about deprecation of certificate bundles from Google HSM.

![image](https://github.com/ebourg/jsign/assets/637767/8ecc75fd-c0f9-488f-a30f-6c775b732796)

So, is there any modification needed in your implementation or in my side?

**Comment by ebourg** [comment, marked as answer]
Don't worry this is just related to the key attestation certificates, it doesn't affect the signing API used by Jsign.

**Reply by carlosspohr** [reply]
Hi @ebourg ,

Thanks for your reply.

I'll validate a new build and sign today and provide a feedback in here.

**Reply by carlosspohr** [reply]
Works normally! 👍
